app/views/users/show/_pre_launch.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <div class="container  pre-launch-container" >
-    <%= show_svg('hactoberfest-mini-logo.svg') %>
+    <%= show_svg('hacktoberfest-mini-logo.svg') %>
```

Comment:
The actual asset file also needs to be renamed

Reply:
I believe it was already named `hacktoberfest` (correctly). I'll check, though.